WebNetSample/Views/Products/Edit.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@using WebNetSample.Entity.Concrete;
+@model Product
+
+<p>
+    <h3 class="text-warning">You are editing <span class="text-danger">@Model.Name</span> product</h3>
+</p>
+
+@using (Html.BeginForm("Update", "Products", FormMethod.Post))
+{
+    <table cellpadding="0" cellspacing="0">
+        <tr>
+            <th colspan="2" align="center">Product Details</th>
+        </tr>
+        <tr>
+            <td>Id: </td>
+            <td>
+                @Html.TextBoxFor(m => m.Id, new { @readonly = "readonly", @Value = @Model.Id })
+            </td>
+        </tr>
+        <tr>
+            <td>Name: </td>
+            <td>
+                @Html.TextBoxFor(m => m.Name, new { @Value = @Model.Name })
+            </td>
+        </tr>
+        <tr>
+            <td>Category: </td>
+            <td>
+                @Html.DropDownListFor(m => m.CategoryId, new List<SelectListItem>
+                { new SelectListItem{Text="Toys", Value="cc1fd238-f590-483e-ab5c-15006f73eb92"}}, "Please select a category")
+            </td>
+        </tr>
+        <tr>
+            <td>Supplier: </td>
+            <td>
+                @Html.DropDownListFor(m => m.SupplierId, new List<SelectListItem>
+                { new SelectListItem{Text="Xyz", Value="aa1e7065-a0cf-467f-942c-cba70a5f7b61"}}, "Please select a supplier")
```

Comment:
Is `Xyz` a really valid value?) Just a curiosity :D

Reply:
Actually, yes. There is no meaning but I named it like that. Sorry for that :)